.k8s/utils/mock-directory.ts

```diff
@@ -0,0 +1,16 @@
+import fs from "fs"
+import path from "path"
+import fsx from "fs-extra"
+import { directory } from "tempy";
+
+export default () => {
+  const cwd = directory();
+
+  fs.symlinkSync(path.join(__dirname, "..", "utils"), `${cwd}/utils`)
+  fsx.copy(path.join(__dirname, "..", "components"), `${cwd}/components`)
+  fs.symlinkSync(path.join(__dirname, "..", "kosko.toml"), `${cwd}/kosko.toml`)
+  fs.symlinkSync(path.join(__dirname, "..", "node_modules"), `${cwd}/node_modules`)
+  fs.symlinkSync(path.join(__dirname, "..", "tsconfig.json"), `${cwd}/tsconfig.json`)
+
+  return cwd
+}
```

Comment:
IO action are better handled with async
You can use `fs-extra` everywhere with `await` on each fs like functions